minecraft/1.8/src/main/java/nova/core/wrapper/mc/forge/v18/wrapper/render/FWSmartModel.java

```diff
@@ -71,17 +87,22 @@ public FWSmartModel() {
 			.stream()
 			.flatMap(
 				model -> {
-					if (model instanceof MeshModel) {
+					if (model instanceof BWBakedModel) {
+						return Stream.concat(((BWBakedModel) model).getGeneralQuads().stream(),
```

Comment:
Get rid of this bit, as `BWBakedModel.clone()` returns an instance of `MeshModel` and this bit doesn’t get called.